product_variant_default_code/tests/test_variant_default_code.py

```diff
@@ -64,7 +64,7 @@ def test_01_check_default_codes(self):
             self.assertEqual(product.default_code, expected_code)
 
     def test_02_check_default_codes_preexistent_mask(self):
-        self.sale_setting.group_product_default_code = 1
+        self.base_setting.group_product_default_code = 1
```

Comment:
Let's force the group because otherwise the config is not catched
```python
self.env.user.groups_id |= self.group_default_code
```

Reply:
done